src/Lotto.js

```diff
@@ -1,18 +1,47 @@
-class Lotto {
+const ErrorBoundary = require('./error/ErrorBoundary');
+const { LOTTO_ERROR_MESSAGE } = require('./constants/lotto');
+
+const Lotto = class extends ErrorBoundary {
   #numbers;
 
   constructor(numbers) {
-    this.validate(numbers);
-    this.#numbers = numbers;
+    super();
+    this.#numbers = this.sortNumbers(numbers);
+    this.validateInput(this.#numbers);
+  }
+
+  sortNumbers(numbers) {
+    return numbers.sort((prevNumber, currNumber) => prevNumber - currNumber);
   }
 
   validate(numbers) {
-    if (numbers.length !== 6) {
-      throw new Error("[ERROR] 로또 번호는 6개여야 합니다.");
+    const isNumberLengthValid = numbers.length === 6;
+    const isNumberNotDuplicated = numbers.length === [...new Set(numbers)].length;
```

Comment:
set을 array로 다시 변환하지 않아도 비교할 수 있을 것 같아요!

```suggestion
    const isNumberNotDuplicated = numbers.length === new Set(numbers).size;
```

Reply:
아 맞다 Set의 size가 있었군요!! 아이코! 감사합니다!!ㅎㅎ